src/screens/Customers/CustomerMain.tsx

```diff
@@ -1,228 +1,231 @@
+  );
 
-    const filterByKeys: string[] = Object.keys(FilterBy);
-    if (filterByKeys.includes(newlySelectedOption)) {
-      setFilterBy(FilterBy[newlySelectedOption as keyof typeof FilterBy]);
-      setFilterLabels(labels[newlySelectedOption as keyof typeof FilterBy]);
-      return;
-    }
-  }
+  const getPaymentTabLabel = () => (
+    <div>
+      <AttachMoneyIcon fontSize='inherit' className={classes.tabIcon} />
+      <Typography className={classes.tabLabel}>Payment</Typography>
+    </div>
+  );
 
   return (
-    <BaseScreen rightIcon="user">
-      <div className={classes.headerWrapper}>
-        <h1 className={classes.title}>Customers</h1>
-      </div>
-      <div className={classes.selectionHeader}>
-        <SearchBar placeholder="Search for a customer" onSearchChange={handleSearchChange} />
-        <FormControl>
-          <Select onChange={handleMenuSelect} multiple value={sortAndFilter} inputProps={{ 'aria-label': 'Without label' }}>
-            <ListSubheader>Sort By</ListSubheader>
-            <MenuItem value="NAME">{SortBy.NAME}</MenuItem>
-            <MenuItem value="METER">{SortBy.METER}</MenuItem>
-            <ListSubheader>Filter By</ListSubheader>
-            <MenuItem value="PAYMENT_STATUS">{FilterBy.PAYMENT_STATUS}</MenuItem>
-            <MenuItem value="METER_STATUS">{FilterBy.METER_STATUS}</MenuItem>
-            <MenuItem value="ACTIVE_STATUS">{FilterBy.ACTIVE_STATUS}</MenuItem>
-          </Select>
-          <div className={classes.rightAlign}><FormHelperText>Sort and Filter</FormHelperText></div>
-        </FormControl>
-      </div>
-      <BaseScrollView>
-        <FormHelperText>{filterLabels[0]}</FormHelperText>
-        {filteredCustomers.map((customer, index) => (
-          <Link key={index} to={`${props.match.url}/customer`} onClick={() => setCurrentCustomerIdInRedux(customer.id)} >
-            <CustomerCard name={customer.name} amount={customer.outstandingBalance} date={getLatestReadingDate(customer)} active={customer.isactive} />
-          </Link>
-        ))
-        }
-        <FormHelperText>{filterLabels[1]}</FormHelperText>
-        {filteredCustomersAlt.map((customer, index) => (
-          <Link key={index} to={`${props.match.url}/customer`} onClick={() => setCurrentCustomerIdInRedux(customer.id)} >
-            <CustomerCard name={customer.name} amount={customer.outstandingBalance} date={getLatestReadingDate(customer)} active={customer.isactive} />
-          </Link>
-        ))
-        }
-      </BaseScrollView>
+    <BaseScreen rightIcon="user" title="Customers" searchAction={handleSearchChange} searchExit={exitSearch}>
+      <TabContext value={value}>
+        <Tabs textColor="primary" indicatorColor="primary" value={value} onChange={changeTab}>
+          <Tab className={classes.tab} label="All" value="0" />
+          <Tab className={classes.tab} label={getMeterTabLabel()} value="1" />
+          <Tab className={classes.tab} label={getPaymentTabLabel()} value="2" />
```

Comment:
It probably makes more sense to have `value` be based off of the `CustomerStatus` enum rather than `0, 1, 2` so it's easy to understand what tab is what value at a glance.